try.go

```diff
@@ -0,0 +1,46 @@
+package lo
+
+func Try(callback func() error) (err bool) {
+	err = true
+
+	defer func() {
+		if r := recover(); r != nil {
+			err = false
+		}
+	}()
+
+	if callback() != nil {
+		err = false
+	}
+
+	return
+}
+
+func TryWithErrorValue(callback func() error) (err bool, errorValue any) {
```

Comment:
I would rather use the following prototype, more respectful of go standards:

```go
func TryWithErrorValue(callback func() error) (errorValue any, ok bool) {
```

(reversed order of returned value)

Reply:
👍

Reply:
Alright, done